wkcuber/pyproject.toml

```diff
@@ -1,49 +1,27 @@
 [tool.poetry]
 name = "wkcuber"
-version = "0.0.0"  # filled by dunamai
+version = "0.0.0"
 description = "Python package to create, cube, and work with WEBKNOSSOS WKW datasets"
 authors = ["scalable minds <hello@scalableminds.com>"]
 readme = "README.md"
 license = "AGPL-3.0"
 repository = "https://github.com/scalableminds/webknossos-libs"
 homepage = "https://docs.webknossos.org/wkcuber"
-include = ["wkcuber/version.py"]  # included explicitly since it's in .gitignore
```

Comment:
Is this no longer required?

Reply:
obsolete since we integrate wkcuber in webknossos package